tornado/httputil.py

```diff
@@ -638,7 +717,8 @@ def _parse_request_range(range_header):
 
     See [0] for the details of the range header.
 
-    [0]: http://greenbytes.de/tech/webdav/draft-ietf-httpbis-p5-range-latest.html#byte.ranges
+    [0]: http://greenbytes.de/tech/webdav/draft-ietf-httpbis-p5-range-latest
+    ...    html#byte.ranges
```

Comment:
Is this valid sphinx markup? Ugh. It's far better to have a line longer than 80 columns than to split a line like this requiring manual repair.